introduction.tex

```diff
@@ -20,22 +20,57 @@ \chapter{Introduction}
 
 \section{Terminology}
 
-A \emph{platform} is a single integrated circuit consisting of one or more
-\emph{components}. Some components may be RISC-V cores, while others may have a different
-function. Typically they will all be connected to a single system bus.
-A single RISC-V core contains one or more hardware threads, called
-\emph{harts}.
-
-\emph{DXLEN} of a hart is its widest supported XLEN, ignoring the current value
-of \Fmxl in \Rmisa.
-
-A \emph{physical address} is directly usable on the system bus.
-
-A \emph{virtual address} is an address as a hart sees it. If there is address
-translation this may be different from the \emph{physical address}. If there is
-no translation then it will be the same.
-
-\subsection{Context}
+\begin{description}[style=nextline]
+    \item[AMO]
+        Atomic Memory Operation.
+    \item[BYPASS]
+        JTAG instruction that selects a single bit data register, also called BYPASS.
+    \item[component]
+        A RISC-V core, or other part of a platform.
+        Typically all components will all be connected to a single system
+        bus.
+    \item[CSR]
+        Control and Status Register.
+    \item[DM]
+        Debug Module (see Section \ref{dm}).
+    \item[DMI]
+        Debug Module Interface (see Section \ref{dmi}).
+    \item[DR]
+        JTAG Data Register.
+    \item[DTM]
+        Debug Transport Module (see Section \ref{dtm}).
+    \item[DXLEN]
+        Debug XLEN, which is the widest XLEN a hart supports, ignoring the
+        current value of \Fmxl in \Rmisa.
+    \item[GPR]
+        General Purpose Register.
+    \item[hart]
+        A hardware thread in a RISC-V core.
+    \item[IDCODE]
+        32-bit Identification CODE, and a JTAG instruction that returns the IDCODE value.
+    \item[IR]
+        JTAG Instruction Register.
+    \item[JTAG]
+        Joint Test Action Group, referring to IEEE 1149.
```

Comment:
```suggestion
        Joint Test Action Group, referring to IEEE 1149.1.
```

Reply:
I consciously didn't include .1, thinking that .7 should also be included. But maybe .7 really is cJTAG only?

Reply:
How about "Refers to work done by IEEE's Joint Test Action Group, described in IEEE 1149.1."

Reply:
That sounds good.  The Debug Spec DTM section only describes an 1149.1 implementation, so it seems correct to refer to that as opposed to "1149".